### PR TITLE
BaSP-MR-188: Fix validations bugs

### PR DESCRIPTION
**Pull request by Gonzarey96**
Fixed validations and reset button bug in all forms.

### CODE REVIEW

File: src/Validations/trainers.jsx [hunk 1]
```diff
@@ -2,58 +2,69 @@ import Joi from 'joi';
...
     .trim()
+    .regex(/^[A-Za-z]+\s?[A-Za-z]*$/)
     .min(3)
     .max(25)
-    .required()
     .messages({
-      'string.pattern.base': 'Name must have only letters',
-      'any.required': 'Name is required',
-      'string.empty': 'Name is required.'
+      'any.required': 'First Name is required',
```
Comment (EmaBaldor) [inline src/Validations/trainers.jsx]: It might be more consistent to match the validation message ('First Name' is required) with the tag label ('Name').

**Review by EmaBaldor — APPROVED** [review]
Good Job!

**Comment by Gonzarey96** [comment]
[[Tech-Debt] Inputs](https://trello.com/c/GhyWgk3i/188-tech-debt-inputs)

**Comment by nazarenapantanetti** [comment]
The alignment of the inputs is successfully corrected but the "reset" button still doesn't work properly. The ticket about it: https://trello.com/c/Q2ThLqCP/237-inputs-bug-fails-reset-button-and-validations

**Pull request merged**